README.md

```diff
@@ -1,22 +1,22 @@
 # ShellHub [![ShellHub](https://circleci.com/gh/shellhub-io/shellhub.svg?style=shield)](https://circleci.com/gh/shellhub-io/shellhub)
 
 ShellHub is a modern SSH server for remotely accessing Linux devices
-via command line (using any SSH client) or web-based user interface.
-It is intended to be used instead of sshd. ShellHub enables teams to
+via command line (using any SSH client) or web-based user interface, 
+designed as an alternative to _sshd_. It enables teams to
 easily access any Linux device behind firewall and NAT.
 
 ![Screenshot](https://github.com/shellhub-io/shellhub-io.github.io/raw/src/docs/img/screenshot.png)
 
 ## Getting started
 
-To start using ShellHub, we recommend that you begin with the
-[Getting started](https://shellhub-io.github.io/getting-started/) section
+To start using ShellHub, it is suggested that you follow the
+[Getting started](https://shellhub-io.github.io/getting-started/) guide 
 in the ShellHub documentation.
 
 ## Contributing
 
-ShellHub is an open source project and we love to receive contributions from
-our community. If you would like to contribute, please read
+ShellHub is an open source project and the community support is very much 
+appreciated. If you would like to contribute, please read
```

Comment:
This last part, I prefer the previous one.

Reply:
A bit cheesy for my taste, but sounds good.